src/tts/magpietts/decoder.cpp

```
@@ -447,6 +830,48 @@ MagpieDecoder::evalCachedPair(
+            if (persistent_runtime_ && persistent_runtime_->eval(
+                                           audio_codes, cond_kv, uncond_kv, cond_result,
+                                           uncond_result, cond_hidden_out, uncond_hidden_out,
+                                           attention)) {
+                return true;
+            }
+            persistent_runtime_.reset();
+            cond_kv.clear();
+            uncond_kv.clear();
+        }
+        catch (const std::exception& e) {
+            fprintf(stderr, "MagpieTTS persistent decoder failed: %s\n", e.what());
```

Clang (14.0.6)

[warning] do not call c-style vararg functions

(cppcoreguidelines-pro-type-vararg)